src/domain/agent/verified-credential/dto/verified.credential.dto.result.ts

```diff
@@ -1,48 +1,30 @@
-import { Field, ObjectType } from '@nestjs/graphql';
-import JSON from 'graphql-type-json';
-
-@ObjectType()
-export class VerifiedCredential {
-  @Field(() => String, {
-    description: 'The type of VC',
-  })
+import { IVerifiedCredential } from '../verified.credential.interface';
+import { VerifiedCredentialClaim } from './verified.credential.dto.claim.result';
+
+export class VerifiedCredential implements IVerifiedCredential {
   type: string;
 
-  @Field(() => String, {
-    description: 'The challenge issuing the VC',
-  })
   issuer: string;
```

Comment:
Issuer and Expiration are two reserved claims. I am not sure whether there is a standard in VC claims, but in JWT claims issuer is always `iss`, and expiration is always `exp`. Please have a look at standard claims notation: https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-token-claims